widgets/price-table/js/pricetable.js

```diff
@@ -1,4 +1,28 @@
 jQuery( function($){
+    $('.ow-pt-columns-atom.sow-equalize-row-heights').each(function () {
```

Comment:
Hmm, yeah that's a good point. However, the current row height calculation is quite specific to the `atom` template layout and its elements. Say we added another template that used a horizontal layout instead of vertical, I think the calculation would be, at least slightly, different. So IMHO it's not worth trying to generalize this before we have another template that would use this.